akka-actor/src/main/scala/akka/actor/Deployer.scala

```diff
@@ -140,11 +140,11 @@ private[akka] class Deployer(val settings: ActorSystem.Settings, val dynamicAcce
       case (key, value: String) ⇒ (key → value)
     }.toMap
 
-  config.root.asScala flatMap {
+  config.root.asScala.map {
     case ("default", _)             ⇒ None
     case (key, value: ConfigObject) ⇒ parseConfig(key, value.toConfig)
     case _                          ⇒ None
-  } foreach deploy
+  }.flatten.foreach(deploy(_))
```

Comment:
That looks like a regression in 2.13. The previous code should compile without changes.

Reply:
https://github.com/scala/bug/issues/11033